includes/class-sensei-dependency-checker.php

```diff
@@ -17,7 +17,7 @@
  * @since 2.0.0
  */
 class Sensei_Dependency_Checker {
-	const MINIMUM_PHP_VERSION        = '7.0';
+	const MINIMUM_PHP_VERSION        = '7.2';
 	const FUTURE_MINIMUM_PHP_VERSION = '7.2';
```

Comment:
I think we could probably remove this constant (and the code around it), no?

because, like, if the user has a version older than PHP 7.2 (let's say that the user decides to force the upgrade through deleting the old plugin and uploading the new one), what will show up?

Reply:
Hey Fernando, thanks for the review! I just did a quick test on PHP 7.0 and it shows the following notice:

<img width="1298" alt="Screenshot on 2022-07-06 at 16-40-12" src="https://user-images.githubusercontent.com/1612178/177563782-380d9437-3e9d-4245-90dd-592dd27966ee.png">

I think it's a good idea to leave it as is, because it also prevents the loading of the rest of the plugin which might break the site once we start adding 7.2 code.

Reply:
Hmm, interesting.

I was talking more about the constant `FUTURE_MINIMUM_PHP_VERSION`, not the `MINIMUM_PHP_VERSION`, but the future one is meant for...well, future releases, you know?

If it doesn't conflict, we can leave it there. I was just worried about this specific scenario. =)

I'll approve the PR, then.

Reply:
Oh, yeah, we can keep the future constant and use it for the next jump. Thanks. 🙂